### DISCUSSION TITLE
facetStats as query parameter

### DISCUSSION THREAD

**Discussion by thijndehaas** [meilisearch/product#658] [Feedback & Feature Proposal]
I'm glad there is now the option for `facetStats`. We would like to use this for our price field, among other things. However, this means that we also get `99999` prices back while we only need the min/max value.

It also seems faster to me to only determine the min/max value instead of building the complete facetDistrubution array.

My idea would be to have a query parameter facetStats . This would allow the existing functionality of automatically adding facetStats in the response to remain.
However, if, for example, a price field is not given as a facet, but only as a facetStats field, this would save a lot of data in the response. And probably more optimization can be achieved within the source code for facetStats without facets.

We have set the `maxValuesPerFacet` to `99999` because we have for example a dropdown filter with brands and we want to show all brand in it. The brands dropdown has a search filter so the user doens't notice the high facetDistrubution amount.
![afbeelding](https://user-images.githubusercontent.com/18503351/231731402-0adfbfa7-ac16-454a-a81b-e0efc90cb468.png)

**Example request:**
```php
$client->index('products')->search('Example', [
  'facets' => ['brand', 'sizes'],
  'facetStats' => ['price']
]);
```
**Example response:**
```json
{
  …
  "estimatedTotalHits": 22,
  "query": "Example",
  "facetDistribution": {
    "brand": {
      "Nike": 2450,
      "Adidas": 713,
      "Puma": 254
    },
    "sizes": {
      "42": 151,
      "43": 231,
      "45": 95
    }
  },
  "facetStats": {
    "sizes": {
      "min": 42,
      "max": 45
    },
    "price": {
      "min": 64,
      "max": 599
    }
  }
}
```

**Comment by oluademola** [comment]
Hello @thijndehaas. Thank you for the feedback. @gmourier & @macraig bringing this to your attention.
